app/assets/javascripts/views/story_view.js

```diff
@@ -513,11 +514,15 @@ module.exports = FormView.extend({
 
     const $storyEstimate = this.$('[data-story-estimate]');
     if ($storyEstimate.length) {
+      var storyEstimateOptions = this.model.point_values().map((option) => {
```

Comment:
We should extract the mapper function to a new method to avoid creating this function on every render, like:

```javascript
var storyEstimateOptions = this.model.point_values().map(this.createStoryEstimateOption);
```

Since you don't use `this` inside the mapper you won't need to bind it 😄